xml/net_bonding.xml

```diff
@@ -292,9 +292,62 @@ KERNEL=="eth*", NAME="eth0"</screen>
    </remark>
   </para>
  </sect2>
+<sect2 xml:id="sec-predictable-naming-scheme">
+  <title>Predictable naming scheme</title>
+  <para>
+    There is a limitation with the persistent network name generator and bonding. When an interface is enslaved to a bond,
+    the MAC address is temporarily overridden and set to match the MAC address of the bond interface.
+    This creates inconsistencies with the <literal>70-persistent-net.rules</literal> file when the rules for enslaved interfaces are based on MAC addresses.
+   </para>
+   <para>
+  If an <systemitem>uevent</systemitem> occurs for an enslaved NIC with an updated MAC address,<systemitem>udev</systemitem> may try to
+  rename it with the first enslaved interface's name, resulting in a temporary name such as <literal>rename4</literal>.
+  Modifying the persistent network name generator to avoid MAC-based rules is not feasible due to the high risk of regression and the necessity of MAC filtering in certain cases
```

Comment:
```suggestion
  Modifying the persistent network name generator to avoid MAC-based rules is not feasible due to the high risk of regressions and the necessity of MAC filtering in certain cases.
```